recipe/bld.bat

```diff
@@ -1,12 +1,31 @@
-MOVE bin\* %LIBRARY_BIN%
-MOVE include\* %LIBRARY_INC%
-MOVE jre %LIBRARY_PREFIX%\jre
-MOVE lib\* %LIBRARY_LIB%
-MOVE src.zip %LIBRARY_PREFIX%\jre\src.zip
-
-:: ensure that JAVA_HOME is set correctly
-mkdir %PREFIX%\etc\conda\activate.d
-echo set "JAVA_HOME_CONDA_BACKUP=%%JAVA_HOME%%" > "%PREFIX%\etc\conda\activate.d\java_home.bat"
-echo set "JAVA_HOME=%%CONDA_PREFIX%%\Library" >> "%PREFIX%\etc\conda\activate.d\java_home.bat"
-mkdir %PREFIX%\etc\conda\deactivate.d
-echo set "JAVA_HOME=%%JAVA_HOME_CONDA_BACKUP%%" > "%PREFIX%\etc\conda\deactivate.d\java_home.bat"
+setlocal EnableDelayedExpansion
+
+xcopy bin\* %LIBRARY_BIN% /s /i /y
+if errorlevel 1 exit 1
+
+xcopy include\* %LIBRARY_INC% /s /i /y
+if errorlevel 1 exit 1
+
+move jre %LIBRARY_PREFIX%\jre
+if errorlevel 1 exit 1
+
+xcopy lib\* %LIBRARY_LIB% /s /i /y
+if errorlevel 1 exit 1
+
+move src.zip %LIBRARY_PREFIX%\jre\src.zip
+if errorlevel 1 exit 1
+
+:: Needed by JNI programs that link against this library.
+copy %LIBRARY_PREFIX%\jre\bin\server\jvm.dll %LIBRARY_BIN%\
+if errorlevel 1 exit 1
+copy %LIBRARY_PREFIX%\jre\bin\*.dll %LIBRARY_BIN%\
+if errorlevel 1 exit 1
```

Comment:
This second copy is probably the wrong approach as it is not targeted enough. Was adding it in as `jvm.dll` said it was missing dependencies, but didn't say which ones. However this is causing new problems, where it is looking for `jvm.cfg` in the wrong location and not finding it. Links to the relevant logs demonstrating these issues in order below.

ref: https://ci.appveyor.com/project/conda-forge/openjdk-feedstock/build/1.0.112/job/8eux8vxmsrb4bcql
ref: https://ci.appveyor.com/project/conda-forge/openjdk-feedstock/build/1.0.115/job/nemxfkj4135v2ies